build_web_compilers/README.md

```diff
@@ -0,0 +1,98 @@
+# build_web_compilers
+
+<p align="center">
+  Web compilers for users of <a href="https://pub.dartlang.org/packages/build"><code>package:build</code></a>.
+  <br>
+  <a href="https://travis-ci.org/dart-lang/build">
+    <img src="https://travis-ci.org/dart-lang/build.svg?branch=master" alt="Build Status" />
+  </a>
+  <a href="https://github.com/dart-lang/build/labels/package%3Abuild_web_compilers">
+    <img src="https://img.shields.io/github/issues-raw/dart-lang/build/package%3Abuild_web_compilers.svg" alt="Issues related to build_web_compilers" />
+  </a>
+  <a href="https://pub.dartlang.org/packages/build_web_compilers">
+    <img src="https://img.shields.io/pub/v/build_web_compilers.svg" alt="Pub Package Version" />
+  </a>
+  <a href="https://www.dartdocs.org/documentation/build_web_compilers/latest">
+    <img src="https://img.shields.io/badge/dartdocs-latest-blue.svg" alt="Latest Dartdocs" />
+  </a>
+  <a href="https://gitter.im/dart-lang/source_gen">
+    <img src="https://badges.gitter.im/dart-lang/source_gen.svg" alt="Join the chat on Gitter" />
+  </a>
+</p>
+
+## Installation
+
+This package is intended to be used as a [development dependency][] for users
+of [`package:build`][] who want to run code in a browser. Simply add the
+following to your `pubspec.yaml`:
+
+```yaml
+dev_dependencies:
+  build_web_compilers:
+```
+
+## Usage
+
+If you are using the autogenerated build script (going through `pub run` instead
+of handwriting a `build.dart` file), then all you need is the `dev_dependency`
+listed above.
+
+If you are using a custom build script, you will need to add the following
```

Comment:
I wonder if we should expose a `List<BuilderApplication>` for this use case?

Reply:
we could consider doing that.... it would likely be more future proof